components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -562,6 +562,27 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
    logical(c_bool)             :: crm_accel_uv
    integer                     :: igstep
    integer                     :: idx_vt_t, idx_vt_q  ! variance transport constituent indices
+
+   ! pointers for crm state data on pbuf
+   real(crm_rknd), pointer :: crm_state_u_wind     (:,:,:,:) ! CRM u-wind component
```

Comment:
Since these are used only to access pbuf fields, I think the naming should follow the pbuf names like we do elsewhere. I.e., `crm_u` instead of `crm_state_u_wind`.

Reply:
will do.

Reply:
done.